src/modules/layout/components/Header.js

```diff
@@ -134,6 +129,10 @@ class Header extends React.PureComponent<PropsType> {
     const { navigation, t, theme } = this.props
     const sharePath = navigation.getParam('sharePath')
 
+    if (!this.props.navigation) {
+      return null
```

Comment:
Is this expected to happen? If not probably a Error should be thrown.

Reply:
I don't know at all why I added these lines... I just removed them again and everything is fine....